tests/flows/erc20_bridge_multihop.go

```diff
@@ -244,6 +247,7 @@ func ERC20BridgeMultihop(network network.Network) {
 
 	// Relay message from SubnetB to SubnetA
 	receipt = network.RelayMessage(ctx, receipt, subnetBInfo, subnetAInfo, true)
```

Comment:
can you add additional comments about how this `receipt` will include the `SendCrossChainmessage` event from A->C since it's not reached its destination, possibly adding to line 260 or here.